src/benchmarking.rs

```diff
@@ -360,27 +360,42 @@ benchmarks! {
 	}: _(RawOrigin::Signed(reporter))
 
 	begin_dispute {
+		// Maximum number of sequential disputed timestamps
+		let d in 1..T::MaxDisputedTimeSeries::get();
```

Comment:
I guess, we need to update this range, as in the first step, we'll get 1 from this and in the below looping, we are setting range as:
```
for i in 1..d - 1
```
so, if in case we get d as 1 then this range will look like this:
```
for i in 1..1-1
```
Which seems incorrect, so it'd be good we set the range in d parameter something like this
```
let d in 3..T::MaxDisputedTimeSeries::get();
```
So that in case of minimum value at least we'll get 1 iteration

Reply:
Makes sense, will fix, thanks!